source/extensions/filters/http/cdn/cdn_loop_parser.cc

```diff
@@ -0,0 +1,357 @@
+#include "extensions/filters/http/cdn/cdn_loop_parser.h"
+
+#include "common/common/statusor.h"
+
+#include "absl/status/status.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cdn {
+namespace CdnLoopParser {
+
+namespace {
+
+// RFC 5234 Appendix B.1 says:
+//
+// ALPHA          =  %x41-5A / %x61-7A   ; A-Z / a-z
+constexpr bool IsAlpha(char c) {
```

Comment:
No, because the functions are locale-aware.  For example, if one runs Envoy in anything other than the `"C"` locale, the `isalpha` will have a different set of allowed "alpha" characters than the RFC.